src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -660,7 +679,7 @@ ApiGlobe.prototype.setRange = function setRange(pRange, isAnimated) {
 
     return this.scene.currentControls().setRange(pRange, isAnimated).then(() => {
         this.scene.notifyChange(1);
-        this.setSceneLoaded().then(() => {
+        return this.setSceneLoaded().then(() => {
             this.scene.currentControls().updateCameraTransformation();
             this.viewerDiv.dispatchEvent(eventRange);
```

Comment:
If I do a setRange(10000), the view position is OK, but the getRange() does not return 10000.

The getRange seems to return old range value. Ex :
- setRange(10000) --> getRange() = 25011386.530042846
- setRange(200000) --> getRange() =10013.066997543165 (almost 10000)